infrastructure/ecs-main/serverless.yml

```diff
@@ -121,11 +121,9 @@ custom:
   cSharpBucketName: docs-csharp-${self:provider.stage}
   cSharpIntgrBucketName: docs-csharp-dev
   goBucketName: docs-go-${self:provider.stage}
-  goIntgrBucketName: docs-go-intgr
   javaBucketName: docs-java-${self:provider.stage}
-  javaIntgrBucketName: docs-java-intgr
   nodeBucketName: docs-node-${self:provider.stage}
-  nodeIntgrBucketName: docs-node-intgr
+  languagesBucketName: docs-languages-${self:provider.stage}
```

Comment:
Do we need to add this bucket to the CDK's [list of buckets](https://github.com/mongodb/docs-worker-pool/blob/f75b3fc36a387b985ab5d5ad2bf633774edc70ef/cdk-infra/utils/buckets.ts#L7)?

Reply:
Great catch!! Thank you. Just added